src/js/personalization/va-profile-beta/actions/index.js

```diff
@@ -71,28 +71,29 @@ function saveFieldHandler(apiRoute, requestStartAction, requestSuccessAction, re
 }
 
 async function sendProfileRequests() {
-  const result = {};
-  const requests = [
+  return [
```

Comment:
Yeah that would be ideal but it isn't a single endpoint because that would probably only make sense for our use case and one of the API goals was to be flexible for others. Right now we're dreaming of GraphQL for this, but a batch request also a good idea